R/Viewer.R

```diff
@@ -238,7 +238,39 @@ odbcPreviewObject.OdbcConnection <- function(connection, rowLimit, table = NULL,
     name <- paste(dbQuoteIdentifier(connection, catalog), name, sep = ".")
   }
 
-  dbGetQuery(connection, paste("SELECT * FROM", name), n = rowLimit)
+  dbGetQuery(connection, odbcPreviewQuery(connection, rowLimit, name ),
```

Comment:
Argument alignment; again http://github.com/lionel-/codegrip is likely the easiest way to see what we want.